lib/assets/SnapshotHelper.swift

```diff
@@ -37,29 +46,51 @@ func snapshot(name: String, waitForLoadingIndicator: Bool = true)
             print("Couldn't detect/set language...")
         }
     }
-    
-    class func snapshot(name: String, waitForLoadingIndicator: Bool = false)
+
+    class func snapshot(name: String, waitForLoadingIndicator: Bool = false, waitForAlertsToBeHidden: Bool = true)
     {
         if (waitForLoadingIndicator)
         {
             waitForLoadingIndicatorToDisappear()
         }
+        if (waitForAlertsToBeHidden)
+        {
+            waitForAlertsToDisappear()
+        }
         print("snapshot: \(name)") // more information about this, check out https://github.com/krausefx/snapshot
-        
+
         let view = XCUIApplication()
         let start = view.coordinateWithNormalizedOffset(CGVectorMake(32.10, 30000))
         let finish = view.coordinateWithNormalizedOffset(CGVectorMake(31, 30000))
         start.pressForDuration(0, thenDragToCoordinate: finish)
         sleep(1)
     }
-    
+
     class func waitForLoadingIndicatorToDisappear()
     {
         let query = XCUIApplication().statusBars.childrenMatchingType(.Other).elementBoundByIndex(1).childrenMatchingType(.Other)
-        
+
         while (query.count > 4) {
             sleep(1)
             print("Number of Elements in Status Bar: \(query.count)... waiting for status bar to disappear")
         }
     }
+
+    class func automaticallyHideAlerts(testCase:XCTestCase) -> NSObjectProtocol
+    {
+        return testCase.addUIInterruptionMonitorWithDescription("Snapshot alert auto hide") { (alert: XCUIElement) -> Bool in
+            print("Alert (\(alert.label)) will be hidden")
```

Comment:
should we add some quotes around the label instead of parens? i.e. `print("Alert \"\(alert.label)\" will be dismissed.`